md-rest-api/src/main/java/com/wipro/ats/bdre/md/rest/ProcessTemplateAPI.java

```diff
@@ -590,18 +528,14 @@ RestWrapper apply(@ModelAttribute("processtemplate")
             // Deleting from existing properties if any properties are deleted in the properties template
 
             LOGGER.debug("checking proctemplate before using it= " + processTemplate.getProcessTemplateId());
-            //         List<ProcessTemplate> processTemplatesForDelete = s.selectList("call_procedures.select-process-template-list", processTemplate);
             List<ProcessTemplate> processTemplatesForDelete = processTemplateDAO.selectPTList(processTemplate.getProcessTemplateId());
             for (ProcessTemplate processTemplateForDelete : processTemplatesForDelete) {
-                //            List<Process> processesForPropDelete = s.selectList("call_procedures.select-process-list-for-template-id", processTemplateForDelete);
                 List<Process> processesForPropDelete = processTemplateDAO.selectPListForTemplate(processTemplateForDelete.getProcessTemplateId());
                 for (Process processForPropDelete : processesForPropDelete) {
-                    //                 List<Properties> propertiesForDelete = s.selectList("call_procedures.select-missing-property-list-for-process-id", processForPropDelete);
                     List<Properties> propertiesForDelete = processTemplateDAO.selectMissingPropListForP(processForPropDelete.getProcessId(), processForPropDelete.getProcessTemplateId());
                     for (Properties propertyForDelete : propertiesForDelete) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS134)